config/config_spineGeHemis.json

```diff
@@ -0,0 +1,82 @@
+{
+	"command": "train",
+	"gpu": 6,
+	"hdf5_path": "../duke/projects/csf_deepseg/spineGeneric/hdf5/SpineGe.hdf5",
+	"csv_path": "../duke/projects/csf_deepseg/spineGeneric/hdf5/SpineGe.csv",
```

Comment:
super

Reply:
could you please move it to ivado/config/